content/docs/components/central-dash/_index.md

```diff
@@ -0,0 +1,5 @@
++++
+title = "Central Dashboard"
+description = "All about central dashboard in the Kubeflow Ecosystem"
```

Comment:
We should remove the word "Ecosystem" here too, as it's not clear what the word means in this context.

Suggested wording:

```
The central user interface (UI) in Kubeflow
```

Reply:
Sounds good!